psql2mysql.py

```diff
@@ -64,17 +64,18 @@ def _query_utf8mb4_rows(self, table, stringColumns):
 
     def scanTablefor4ByteUtf8Char(self, table):
         stringColumns = self.getStringColumns(table)
-        LOG.info("Scanning Table %s (columns: %s) for problematic UTF8 "
+        LOG.debug("Scanning Table %s (columns: %s) for problematic UTF8 "
                  "characters", table.name, stringColumns)
         rows = self._query_utf8mb4_rows(table, stringColumns)
-        incompatible_rows = []
+        incompatible = []
+        primary = list(table.primary_key)[0].name
```

Comment:
Can we include all primary keys here? IIRC at least some tables have use multiple columns for their primary key.

Reply:
Do you happen to know which table to look at?

Reply:
OK, check now